articles/azure-functions/functions-scale.md

```diff
@@ -125,6 +125,8 @@ In the consumption and premium plans, the scale controller automatically scales
 
 Function code files are stored on Azure Files shares on the function's main storage account. When you delete the main storage account of the function app, the function code files are deleted and cannot be recovered.
 
+There is no downside or impact to having multiple apps share the same consumption plan. The only aspect that the consumption plan determines is the region. Sharing it between apps has no impact on resiliency, scalability, or reliability of each individual function apps.
```

Comment:
```suggestion
There is no downside or impact to having multiple apps assigned to the same consumption plan. The only aspect that the consumption plan determines is the region. Assigning multiple apps to the same consumption plan has no impact on resilience, scalability, or reliability of each app.
```

Reply:
@danielstocker I need to make some more extensive updates to your additions based on my discussions this am with @jeffhollan. I have cherry-picked your commits into my own fork in the private repo and will do ths fix up there (you will still get credit for helping with the article). @jeffhollan I will have you review my updates, which should go live on 5/6.